Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:16
+
+WORKDIR /usr/src/app
+
+COPY package.json ./
```

Comment:
If our `WORKDIR` is `/usr/src/app`, shouldn't we copy our `package.json` into `/usr/src/app/`?

Reference: [vemuruadi/graphql-node-express-mongo-docker/blob/master/Dockerfile](https://github.com/vemuruadi/graphql-node-express-mongo-docker/blob/master/Dockerfile)

Reply:
Yes, it will be copied in the `/usr/src/app` folder. We do not need to specifically write that after defining the `WORKDIR`.
1. [NodeJS Docker Guide](https://nodejs.org/en/docs/guides/nodejs-docker-webapp/#creating-a-dockerfile)
2. [WORKDIR](https://www.educative.io/answers/what-is-the-workdir-command-in-docker)